src/studio/ui/EditorPane.java

```diff
@@ -1,13 +1,11 @@
 package studio.ui;
 
 import org.fife.ui.rsyntaxtextarea.RSyntaxTextArea;
-import org.fife.ui.rtextarea.ConfigurableCaret;
 import org.fife.ui.rtextarea.RTextScrollPane;
 import studio.kdb.Config;
 import studio.ui.rstextarea.RSTextAreaFactory;
 
 import javax.swing.*;
-import javax.xml.soap.Text;
 import java.awt.*;
```

Comment:
Thanks for spotting unused imports! I have to update my Idea's settings to organise imports on commits. And also I'd love to add automation for checking code styling

Reply:
no worries - it was breaking under java 11...

I'm working on getting all of your changes in finos/kdbStudio - we can see how to add checkstyle at some point...

Reply:
https://github.com/finos/kdb-studio/pull/60